src/index.jsx

```diff
@@ -17,7 +18,10 @@ const AppWrapper = () => (
     <ConnectedRouter history={history}>
       <CookiesProvider>
         <Router>
-          <App />
+          <Switch>
+            <Route path="/login" component={LoginPage} />
+            <App />
```

Comment:
I believe `<App />` should have a wildcard path here (i.e., `<Route path="" component={App} />`) as I believe `App` would otherwise be rendered regardless of the route.